tests/io/v0_0_0/test_http_backend.py

```diff
@@ -83,7 +83,7 @@ def test_tiff(self):
                     "ch": 0,
                 },
                 "file": "tile.tiff",
-                "format": "tiff",
```

Comment:
please revert this change.

Rationale for the trailing commas: if something is added, then we don't affect the line above.